### ISSUE TITLE
React 15 uncontrolled to controlled input warning

### ISSUE THREAD

**Issue by fbarrailla** [mbrookes/formsy-material-ui#79]
When I begin to edit a TextField, I get this warning from `React@15.0.1`

`Warning: TextField is changing a uncontrolled input of type text to be controlled. 
Input elements should not switch from uncontrolled to controlled (or vice versa). 
Decide between using a controlled or uncontrolled input element for the lifetime of the component. 
More info: https://fb.me/react-controlled-components`

Related topics
[Warn when an input switches between controlled and uncontrolled #5864](https://github.com/facebook/react/pull/5864)
[A small breaking change in React v15 impacts redux-form compatibility](https://github.com/erikras/redux-form/issues/703)

**Comment by devdlabs** [comment]
I think it is related to formsy issue

**Comment by mbrookes** [comment]
Yeah, I noticed this too, and haven't had a chance to get to the bottom of it. Good news is, it looks like formsy will get a new release soon, so if it is formsy, it can be fixed there.

**Comment by mbrookes** [comment]
Fixed by formsy-react 0.18.0 (just released) included in formsy-material-ui 0.4.0 (also jest released). 😄

**Comment by mb-copart** [comment]
I am seeing the same warning, even when not using formsy-react. 
```
"react": "^15.0.0",
"material-ui": "^0.15.0-beta.1",
"redux": "^3.3.1",
"react-redux": "^4.4.1"
```
